Backpack-SwiftUI/Calendar/Classes/BPKCalendar.swift

```diff
@@ -50,7 +51,7 @@ public struct BPKCalendar<DayAccessoryView: View>: View {
         calendar: Calendar,
         validRange: ClosedRange<Date>,
         initialMonthScroll: MonthScroll? = nil,
-        calendarAccessibilityConfiguration: CalendarAccessibilityConfiguration,
+        calendarAccessibilityConfiguration: CalendarAccessibilityConfiguration = .init(),
```

Comment:
It's not that it shouldn't be mandatory, it should.
The thing is that the API should not allow for the component to be in an invalid state. ie, this API allows for me to not set accessibiltiy, where that should never be the case.
What the api should allow for, is so that the correct accessibility configurations are asked for the right selection mode.
example, i'm configuring a calendar with single selection only, it should not ask for a reanged selection a11y config, as its unnecesary. While also forcing the client to pass a11y configuration for single selection, as we take a11y as a requirement. it should not be possible to instantiate a calendar without setting it.